skyagi-web/src/routes/room/new/+page.svelte

```diff
@@ -7,6 +7,7 @@
     import { browser } from '$app/environment';
     export const characterData = data.agents
     export const modelData = data.models
+    export const userId = data.userId
```

Comment:
You don't need to 'export' the variable if its value will not be assigned from outside.

Reply:
fixing now tyyyyy!